### ISSUE TITLE
Add support for the auto animation of source code

### ISSUE THREAD

**Issue by HuangBoHong** [yjwen/org-reveal#449]
As #443 mentioned, the `data-id` property used by auto animation could be added through ` :code_attribs`. But if the `data-id` property is placed in `<code>`, the animation could be weird like this:
![Peek 2021-08-24 16-53](https://user-images.githubusercontent.com/11132087/130589439-a268be60-892e-422a-91bd-22d428fe9199.gif)
And the animation works fine providing the `data-id` is placed in `<pre>` manually:
![Peek 2021-08-24 16-51](https://user-images.githubusercontent.com/11132087/130589859-caf709a8-fbd2-4fb1-97f2-7de850760649.gif)
 So could there be any support for modifying the properties of `<pre>` or manipulating auto animation directly ?  Thanks. 😃

**Comment by yjwen** [comment]
Are you using highlight plugin for source code highlights?

**Comment by HuangBoHong** [comment]
> Are you using highlight plugin for source code highlights?

Yes. Auto animation for source code will not work if code is highlighted via htmlize..

**Comment by HuangBoHong** [comment]
> Fixed. Auto-animation support is updated. Please check https://github.com/yjwen/org-reveal#auto-animate

That works, thanks to your great job!

**Comment by HuangBoHong** [comment]
> Fixed. Auto-animation support is updated. Please check https://github.com/yjwen/org-reveal#auto-animate

In addition, according to https://revealjs.com/auto-animate/#example%3A-animating-between-code-blocks, `data-line-numbers` should be manually enabled via `code_attribs`. Could you mention it in README, or enable it by default ?

**Comment by yjwen** [comment]
It is already documented in README at section [Using highlight.js](https://github.com/yjwen/org-reveal#using-highlightjs)

**Comment by HuangBoHong** [comment]
> Fixed. Auto-animation support is updated. Please check https://github.com/yjwen/org-reveal#auto-animate

I found the support for auto animation broken again due to the empty property of `<pre>`. 
![img](https://user-images.githubusercontent.com/11132087/131883478-f1b3ef25-0be9-4eed-9885-5a34c77fa63b.png)
Could you fix it ?

**Comment by yjwen** [comment]
My local test didn't find anything wrong. Here is my sample org.
```
#+REVEAL_PLUGINS: (highlight)
# Both OK with or without highlight

* Heading 1
  :PROPERTIES:
  :REVEAL_EXTRA_ATTR: data-auto-animate
  :END:
  
  #+ATTR_REVEAL: :data_id foo
  #+BEGIN_SRC js
    let index = 1
  #+END_SRC


* Heading 2
  :PROPERTIES:
  :REVEAL_EXTRA_ATTR: data-auto-animate
  :END:


  #+ATTR_REVEAL: :code_attribs data-line-numbers='1|3' :data_id foo
  #+begin_src js
    let index = 1
    let value = 2
  #+end_src
```
Could you please show me a minimum org that can reproduce the problem?

**Comment by HuangBoHong** [comment]
> My local test didn't find anything wrong. Here is my sample org.
> 
> ```
> #+REVEAL_PLUGINS: (highlight)
> # Both OK with or without highlight
> 
> * Heading 1
>   :PROPERTIES:
>   :REVEAL_EXTRA_ATTR: data-auto-animate
>   :END:
>   
>   #+ATTR_REVEAL: :data_id foo
>   #+BEGIN_SRC js
>     let index = 1
>   #+END_SRC
> 
> 
> * Heading 2
>   :PROPERTIES:
>   :REVEAL_EXTRA_ATTR: data-auto-animate
>   :END:
> 
> 
>   #+ATTR_REVEAL: :code_attribs data-line-numbers='1|3' :data_id foo
>   #+begin_src js
>     let index = 1
>     let value = 2
>   #+end_src
> ```
> 
> Could you please show me a minimum org that can reproduce the problem?

Sorry for misreporting the problem. I misspelled `data_id` as `data-id`. Thanks for your reply anyway.
